.github/workflows/ci.yml

```diff
@@ -0,0 +1,95 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+
+env:
+  SECRET_KEY: topsecret123
+
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    container: python:3.7-buster
+
+    services:
+      postgres:
+        image: postgres:10
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_USER: kukkuu
+          POSTGRES_PASSWORD: kukkuu
+          POSTGRES_DB: kukkuu
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Cache pip packages
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-pip-modules
+        with:
+          path: ~/.pip-cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/requirements-dev.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt -r requirements-dev.txt codecov && \
+          apt-get update && apt-get install -y build-essential
```

Comment:
We had to add this to be able to install `gdal-bin` which was required for some geospatial stuff, so if you're not ussing that you could (1) get rid of this line and (2) use the "default" [python setup](https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python#starting-with-the-python-workflow-template) rather than the ubuntu-python docker image (same way it's done on the `coding-style` task).